backend/app/alembic/versions/b0c92ff46404_add_enabled_field_to_prompts_table.py

```
@@ -0,0 +1,29 @@
+"""add enabled field to prompts table
+
+Revision ID: b0c92ff46404
+Revises: a5ec0a34da72
+Create Date: 2025-05-26 22:35:15.808303
+
+"""
+from alembic import op
```

Ruff (0.11.9)

`alembic.op` imported but unused

Remove unused import: `alembic.op`

(F401)